File: programme/lowercase-to-uppercase/LowerCaseToUppercase.java
// Write a programme to convert lowercase string to uppercase string
import java.util.Scanner;

public class LowerCaseToUppercase {

  private static String lowerCaseToUpperCase(String lowercase) {
    return lowercase.toUpperCase();
  }

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.print("Enter string : ");
    String input = scanner.next();
    System.out.println(
      "\nString to uppercase : " + lowerCaseToUpperCase(input)
    );
    scanner.close();
  }
}

File: programme/lowercase-to-uppercase/lowercase-to-uppercase.c
// Write a programme to convert lowercase string to uppercase
#include <stdio.h>
#include <string.h>

int main()
{
  char input[100];
  int i;
  printf("Enter a string : ");
  gets(input);
  for (i = 0; input[i] != '\0'; i++)
  {
    if (input[i] >= 'a' && input[i] <= 'z')
    {
      input[i] = input[i] - 32;
    }
  }
  printf("\nString in Upper Case : %s\n", input);
  return 0;
}
